test/delayed_queue_test.rb

```diff
@@ -265,30 +259,30 @@
     end
   end
 
-  test 'enqueue_next_item picks one job' do
+  test 'next_delayed_item picks one job' do
     t = Time.now + 60
 
     Resque.enqueue_at(t, SomeIvarJob)
     Resque.enqueue_at(t, SomeIvarJob)
-    Resque::Scheduler.enqueue_next_item(t)
+    Resque.next_delayed_item(before: t)
     assert_equal(1, Resque.delayed_timestamp_peek(t, 0, 3).length)
   end
 
-  test 'enqueue_delayed_items_for_timestamp creates jobs ' \
-       'and empties the delayed queue' do
-    t = Time.now + 60
+  # test 'enqueue_delayed_items_for_timestamp creates jobs ' \
```

Comment:
this method was removed entirely so the test is irrelevant.

Reply:
so can we remove this test? is all the tested behaviour tested somewhere else?